tests/ixia/ecn/files/helper.py

```diff
@@ -99,31 +90,26 @@ def run_ecn_test(api,
     capture_config = __config_capture_ip_pkt(testbed_config=testbed_config, port_id=port_id)
 
     """ Generate traffic config """
-    flows = __gen_traffic(testbed_config=testbed_config,
-                          port_id=port_id,
-                          pause_flow_name=PAUSE_FLOW_NAME,
-                          data_flow_name=DATA_FLOW_NAME,
-                          prio=lossless_prio,
-                          data_pkt_size=pkt_size,
-                          data_pkt_cnt=pkt_cnt,
-                          data_flow_delay_sec=DATA_START_DELAY_SEC,
-                          exp_dur_sec=EXP_DURATION_SEC,
-                          prio_dscp_map=prio_dscp_map)
-
-    """ Tgen config = testbed config + flow config + capture config"""
-    config = testbed_config
-    config.flows = flows
-    config.captures = capture_config
+    __gen_traffic(testbed_config=testbed_config,
```

Comment:
Please review description of this function. It doesn't return `config` anymore, rather mutates the one that is passed as an argument.

Reply:
updated